packages/mobile/src/screens/profile-screen/ProfileHeader/ViewAllSupportingTile.tsx

```diff
@@ -51,31 +53,29 @@ export const ViewAllSupportingTile = () => {
     'user_id',
     'supporting_count'
   ])
-  const supportingForProfile: SupportingMapForUser =
-    useSelector((state: CommonState) =>
-      getOptimisticSupportingForUser(state, user_id)
-    ) || {}
-  const rankedSupportingIds = Object.keys(supportingForProfile)
-    .sort((k1, k2) => {
-      const amount1BN = stringWeiToBN(
-        supportingForProfile[k1 as unknown as ID].amount
-      )
-      const amount2BN = stringWeiToBN(
-        supportingForProfile[k2 as unknown as ID].amount
-      )
-      return amount1BN.gte(amount2BN) ? -1 : 1
-    })
-    .map((k) => supportingForProfile[k as unknown as ID])
-    .map((s) => s.receiver_id)
-  const rankedSupporting = useSelector((state) => {
-    const usersMap = getUsers(state, { ids: rankedSupportingIds })
-    return rankedSupportingIds.map((id) => usersMap[id]).filter(Boolean)
-  })
+
+  const rankedSupportingList = useRankedSupportingForUser(user_id)
```

Comment:
would it make sense to merge useRankedSupportingForUser with the other proxySelector, so its just one?

Reply:
was wondering the same

Reply:
The reason I didn't merge them is that the components consuming the `Supporting[]` lists handle them differently. Calculating the ranked list of `Supporting[]` objects is where the common code path ends. Though we could probably update them all to just accept a list of `User` objects as calculated here.

Reply:
Gotcha, makes sense that there isn't common code, im wondering if having a single selector would just be more performant for this one? Ultimately I think this is minor but though

Reply:
If it helps, we limit the list that's being sorted to 9 items...something I didn't learn until right before I opened this PR 😆